docs/development/CI/cscs-ci.md

```diff
@@ -0,0 +1,35 @@
+# CSCS-CI Configuration
+
+## Initial Setup
+
+Follow the steps in the [CSCS-CI documentation](https://gitlab.com/cscs-ci/ci-testing/containerised_ci_doc). You will require the help of someone who can register the project to be allowed to run CI on CSCS machines.
+
+## Current Configuration
+
+The configuration can be viewed and changed on the [setup page](https://cicd-ext-mw.cscs.ch/ci/setup_ci) after logging in with the credentials for "cscs-ci setup for gt4py", which can be found on the internal credentials store in the GT4Py folder.
+
+### Whitelisted Users
+
+The core GT4Py developers should be whitelisted. This means that pipelines run on their PRs automatically and they can run them on other contributor's pipelines by commenting "cscs-ci run [pipeline name]".
```

Comment:
Just a question: what is the current state of the whitelist? Which developers are already there? If the current core developers are already in the whitelist I would rephrase it slightly to emphasize that "**new**" core developers should be added to the whitelist.

Reply:
Good point. Right now it's everyone who is in the core gt4py developers group. I didn't want to put any information that will go out of sync very quickly. But adding "new" is a good solution.